### PR TITLE
Gives low tech requirements for 4 roundstart mech items

### PR DESCRIPTION
**Pull request by ghost**
The exosuit fabricator has some very high tech level items buildable round start, most notably the exosuit syringe gun. The syringe gun cannot even be used in an odysseus at round start because the circuit boards for it need higher tech levels. This is the only buildable round start item which can give almost maxed levels on 2 different research branch. This pull removes some of these high tech level items from the round start list and gives them low level requirements which are comparable to other similar mech items. 

This is my first pull so please tell me if something needs to be fixed. I'm not still quite sure how these things work but at least everything worked perfectly when I tested this locally.

### CODE REVIEW

File: code/modules/research/designs/mecha_designs.dm [hunk 1]
```diff
@@ -373,4 +373,48 @@
...
+/datum/design/mech_syringe_gun
+	name = "Exosuit Medical Equipment (Syringe Gun)"
+	desc = "Equipment for medical exosuits. A chem synthesizer with syringe gun. Reagents inside are held in stasis, so no reactions will occur."
+	id = "mech_syringe_gun"
+	build_type = MECHFAB
+	req_tech = list("magnets" = 3,"biotech" = 3)
+	build_path = /obj/item/mecha_parts/mecha_equipment/syringe_gun
+	materials = list(MAT_METAL=3000,MAT_GLASS=2000)
+	construction_time = 200
+ 	category = list("Exosuit Equipment")
```
Comment (tkdrg) [inline code/modules/research/designs/mecha_designs.dm]: We spent a while trying to figure this out on coderbus, but it turns out you left a stray space in the beginning of this line. Delete it (leaving only the tab) and it should compile fine.

Comment (LatD) [inline code/modules/research/designs/mecha_designs.dm]: Now it worked, thanks. I didn't notice a single space had appeared there.

**Comment by Lumipharon** [comment]
This was brought up before, and was shotdown because mech syringe guns are needed to reach certain levels.
I'm not sure if something else can give bio 5 now, but if there isn't, you're making a gap in the R&D tree.
Also you could just like, make some req tech levels for this or some shit.

**Comment by LatD** [comment]
This pull is just making the syringe guns not buildable at round start. The levels which they give are kept the same. So you can still go from bio and electro 3 to 5 with these but not from 1 to 5 like until now.

Keeping the syringe guns at round start but lowering the levels they give is another option but like you said, they are used in almost all guides and are pretty necessary for getting bio and electro 5 from 4 and this made me just raise the levels needed to build them.

**Comment by Dorsisdwarf** [comment]
@lumipharon

>This was brought up before, and was shotdown because mech syringe guns are needed to reach >certain levels.
>I'm not sure if something else can give bio 5 now, but if there isn't, you're making a gap in the R&D tree.
>Also you could just like, make some req tech levels for this or some shit.

You win the 2015 Didn't Read Any Of The PR award.

**Comment by Incoming5643** [comment]
slime cores have bio 4 and can be used instead of mech syringe guns. They're also available fairly quickly (the xenobiologist is gonna get pissed if you kill one of the roundstart grays though.

**Comment by Lumipharon** [comment]
Whoops, I can tell the difference between requirement and origin.

**Comment by Fox-McCloud** [comment]
Ewww, no :-1:

**Comment by Dorsisdwarf** [comment]
I say :+1: because it means they have to rewrite the R&D guides and it's always mildly amusing when they have to reconfigure their cheatsheets

**Comment by Fox-McCloud** [comment]
>because it means they have to rewrite the R&D guides and it's always mildly amusing when they have to reconfigure their cheatsheets

ie: intentionally pissing of a group of players purely to piss them off is a good reason for this PR.-> seems like a good balancing tactic.

**Comment by Lumipharon** [comment]
>my maximum lazy guide remains completely unchanged

Feels good, fuck all those people that do triple syinge guns anyway.

**Comment by LatD** [comment]
I'd compare the current syringe guns to a medicine in medbay which could cure almost all damage types and some of these damage types better than anything else, the medicine would be easier to make than any other medicine and it could be made at round start. Of course there would be people opposing this change but really, these need to get nerfed.

All the guides that use 3 syringe guns at round start seem to be over 2 years old anyway. We've got lots of new items this year, including a new tier of upgradable items. Maybe this will also get rid of the really old guides too.

**Comment by Razharas** [comment]
This will be merged in 3 hours

**Pull request merged**